styles.css

```diff
@@ -28,12 +29,19 @@ aside{
   padding:1em;
   border: 5px solid #aaa;
 }
+.clearfix:after{
+  clear:both;
+  content: "";
+  visibility: hidden;
+  display: block;
+  height: 0;
+}
 img{
   max-width:100%;
 }
 
 @media(max-width:600px){
   .main{
-    background:red;
-  }
+    flex-direction: column;
+    }
```

Comment:
For Question #7, you'll need to make a use of Flexbox's `order` property in order to re-arrange the order of elements when the page is viewed on a mobile device.